public class exception{
	public static void main(String[] args) {
		int a = 10,ans = 0;
		try{
			ans = a/ans;
		}
		catch(Exception e){
			System.out.println("Demonsteration Should not be Zero");
		}
	}
}